website/static/js/pages/project-dashboard-page.js

```diff
@@ -50,7 +50,8 @@ if ($comments.length) {
 
 // Initialize CitationWidget if user isn't viewing through an anonymized VOL
 if (!ctx.node.anonymous && !ctx.node.isRetracted) {
-    new CitationList('#citationList');
+    var citations = ctx.node.alternativeCitations;
```

Comment:
We generally try to avoid using contextVars. Instead, can you add the alternative citations to the response body for the /v1/project/<pid> ... endpoint? That way, the alternative citations will be accessible from `data` here.